File: uri 1010.c
#include <stdio.h>
int main()
{
    int c1,u1,c2,u2;
    float p1,p2,Total;
    scanf("%d%d%f",&c1,&u1,&p1);
    scanf("%d%d%f",&c2,&u2,&p2);
    Total = (u1*p1)+(u2*p2);
    printf("VALOR A PAGAR: R$ %0.2f\n",Total);
    return 0;
}

File: uri 1010.java
import java.util.Scanner;
 
public class Main {
 
    public static void main(String[] args)  {
 
        Scanner input = new Scanner(System.in);
        int c1,u1,c2,u2;
        double p1,p2,sumaiya;
         c1 = input.nextInt();
         u1 = input.nextInt();
         p1 = input.nextDouble();
         c2 = input.nextInt();
         u2 = input.nextInt();
         p2 = input.nextDouble();
         sumaiya = ((u1*p1)+(u2*p2));
         System.out.printf("VALOR A PAGAR: R$ %.2f\n",sumaiya);        
         
    }
 
}